regensdk.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Run in root of repo
+rm -rf hl2sdk-*
+git clone https://github.com/alliedmodders/hl2sdk --branch tf2     --single-branch hl2sdk-tf2  --depth 1
+git clone https://github.com/alliedmodders/hl2sdk --branch sdk2013 --single-branch hl2sdk-2013 --depth 1
+
+# Changes from SDK2013:
+# copy libsteam_api.so
+cp ./hl2sdk-2013/lib/public/linux32/libsteam_api.so ./bin/libsteam_api.so
+# Fix conflicting return type with GetPlayerMins/GetPlayerMaxs
+# https://github.com/alliedmodders/hl2sdk/pull/99
+rm  ./hl2sdk-tf2/game/shared/gamemovement.h
+# permalink to this commit
+curl https://raw.githubusercontent.com/sapphonie/hl2sdk/10934b8c9edf900850b1d85f1b9eff90addc0084/game/shared/gamemovement.h -o ./hl2sdk-tf2/game/shared/gamemovement.h
```

Comment:
Is this meant to be part of the PR? I would not include it as it adds a dependency on your repository.

Reply:
You could (and should) just fork the hl2sdk-2013 repo sourcemod uses, checkout the tf2 branch, and add the updated steamworks http headers to it (https://partner.steamgames.com/downloads/steamworks_sdk.zip), and then add the single fix i pr'd to alliedmodders so it'll compile. i have it sitting at 

https://github.com/sapphonie/hl2sdk/tree/tftrue-tf2, you can compare it to the alliedmodders repo @
https://github.com/alliedmodders/hl2sdk/compare/tf2...sapphonie:tftrue-tf2

Reply:
you can also just remove this script, it's something i added before i forked it myself for tftruer, lol

Reply:
Seems like the fix was merged now. Are the updated steamworks http headers needed?

Reply:
Yes